apps/dashboard/src/app/search/search.component.ts

```diff
@@ -17,17 +18,26 @@ const searchResultsId = 'search-results';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class SearchComponent implements OnDestroy, OnInit {
-  searchWidget = this.sdk.currentKb.pipe(
-    distinctUntilKeyChanged('id'),
+  enabledAnswer =
+    !localStorage.getItem(ENABLE_GENERATIVE_ANSWER) || localStorage.getItem(ENABLE_GENERATIVE_ANSWER) === 'true';
+  private _answerToggle = new BehaviorSubject<boolean>(this.enabledAnswer);
+  searchWidget = combineLatest([this._answerToggle, this.sdk.currentKb.pipe(distinctUntilKeyChanged('id'))]).pipe(
     tap(() => {
       document.getElementById(searchWidgetId)?.remove();
     }),
-    switchMap((kb) =>
+    switchMap(([toggle, kb]) =>
```

Comment:
toggle doesn't mean anything on this context, you should put something more meaningful like `toggleAnswerEnabled` or so